lib/Model/RecentPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace OCA\Collectives\Model;
+
+class RecentPage {
```

Comment:
How about adding a constructor that allows to pass all the values to `new RecentPage()`? We'll probably only fill the `RecentPage` objects once in most cases anyway, no?

Reply:
The idea crossed my mind too, but I dislike huge constructors. They make it also a bit less flexible.

Reply:
Note to myself: chain the methods :)